CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Bug Fixes:
 - Fix triggers of "Bad CMake Executable" error message. [#2368](https://github.com/microsoft/vscode-cmake-tools/issues/2368)
 - Don't ignore empty cache string variables when configuring from presets. [#1842](https://github.com/microsoft/vscode-cmake-tools/issues/1842)
 - Fix active build configuration warning coming from CppTools. [#2353](https://github.com/microsoft/vscode-cmake-tools/issues/2353)
+- Allow a way to run ctests in parallel by setting `cmake.ctest.allowParallelJobs` to `true`. [#3271](https://github.com/microsoft/vscode-cmake-tools/pull/3271)
```

Comment:
Just looking at the other examples here, we typically link to the issue instead of the PR.

Reply:
i'll fix that

Reply:
done